src/models/deliverable/deliverableUnit.ts

```diff
@@ -0,0 +1,27 @@
+import { DELIVERABLE_ACTIONS } from "../../components/Deliverable/constants";
+
+export interface IDeliverableUnit {
+	id?: number;
+	name: string;
+	description?: string;
+	code: string;
+	deliverableCategory?: number | string;
+	unit_type: number | string;
+	prefix_label: number | string;
+	suffix_label: number | string;
+	organization?: number | string;
+}
+
+export type DeliverableUnitProps =
```

Comment:
some of the keys are common in obth cases. Can we initialize the monly once?

Reply:
done